.rubocop.yml

```diff
@@ -155,6 +158,9 @@ Naming/HeredocDelimiterNaming:
 Naming/PredicateName:
   Enabled: false
 
+Naming/RescuedExceptionsVariableName:
```

Comment:
This one wants all rescued errors to be named `e` instead of something like `error` or `api_error` or whatever. Felt like an unnecessary update.

Reply:
Looks good.